modules/gdscript/gdscript_parser.cpp

```diff
@@ -4831,6 +4875,45 @@ bool GDScriptParser::warning_annotations(AnnotationNode *p_annotation, Node *p_t
 #endif // DEBUG_ENABLED
 }
 
+bool GDScriptParser::warning_ignore_region_annotations(AnnotationNode *p_annotation, Node *p_target, ClassNode *p_class) {
+#ifndef DEBUG_ENABLED
+	// Only available in debug builds.
```

Comment:
I'd flip this condition for readability, i.e.:
```cpp
#ifdef DEBUG_ENABLED
...
#else
...
#endif
```

Reply:
See https://github.com/godotengine/godot/pull/76020#discussion_r1190040033. Funny, we've come full circle (this was changed in another PR, then this one was rebased).